### ISSUE TITLE
Web: Conditional style with empty branch does not work

### ISSUE THREAD

**Issue by SebastianAigner** [JetBrains/compose-jb#641]
Consider the following snippet:

```kotlin
renderComposable(rootElementId = "root") {
    var isStruck by remember { mutableStateOf(false) }
    Span(style = {
        if(isStruck) {
            property("text-decoration", StylePropertyValue("line-through"))
            color("red")
        } else {
            // Empty else branch does not work!
        }
    }) {
        Text("Strike me!")
    }
    Button(attrs = { onClick { isStruck = !isStruck } }) { Text("Ok (currently $isStruck)") }
}
```

With an empty else branch, clicking the button changes the state correctly, but does not change the style.

https://user-images.githubusercontent.com/2178959/117163366-dcd35f00-adc3-11eb-85f4-8789ea2cc0ae.mov

Explicitly adding the else case makes the style change work:

```kotlin
if(isStruck) {
    property("text-decoration", StylePropertyValue("line-through"))
    color("red")
} else {
    // Explicitly negate the property
    property("text-decoration", StylePropertyValue("none"))
}
```

Now, styles are updated properly:

https://user-images.githubusercontent.com/2178959/117163650-2623ae80-adc4-11eb-9666-f3b418ca8f5a.mov

**Comment by eymar** [comment]
Thanks for reporting this. It seems we don't clear the style properties. 
I'll fix this

**Comment by eymar** [comment]
We published a new version - `0.0.0-web-dev-12`. It contains the fix.

**Comment by okushnikov** [comment]
Please check the <a href="https://youtrack.jetbrains.com/issue/CMP-6426">following ticket</a> on YouTrack for follow-ups to this issue. GitHub issues will be closed in the coming weeks.
